pipelines/optimus/test_optimus_full_datasets/4k_panT_cells/submit.sh

```diff
@@ -0,0 +1,12 @@
+skylab_path="/Users/nbarkas/work/optimus/runs_Mar19/skylab"
```

Comment:
Did you mean to commit these files? I'm not sure what the point of committing this file (and `log.txt`, `options.json`, `labels.json`) is. Isn't providing the input.json sufficient for users?